cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/network/DefaultNetworkChannel.java

```diff
@@ -0,0 +1,134 @@
+package de.dytanic.cloudnet.driver.network;
+
+import com.google.common.base.Preconditions;
+import de.dytanic.cloudnet.common.concurrent.CompletableTask;
+import de.dytanic.cloudnet.common.concurrent.ITask;
+import de.dytanic.cloudnet.common.document.gson.JsonDocument;
+import de.dytanic.cloudnet.driver.network.def.internal.InternalSyncPacketChannel;
+import de.dytanic.cloudnet.driver.network.protocol.DefaultPacketListenerRegistry;
+import de.dytanic.cloudnet.driver.network.protocol.IPacket;
+import de.dytanic.cloudnet.driver.network.protocol.IPacketListenerRegistry;
+import de.dytanic.cloudnet.driver.network.protocol.chunk.ChunkedPacketBuilder;
+import de.dytanic.cloudnet.driver.network.protocol.chunk.ChunkedQueryResponse;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLong;
+
+public abstract class DefaultNetworkChannel implements INetworkChannel {
+
+    private static final AtomicLong CHANNEL_ID_COUNTER = new AtomicLong();
+
+    private final long channelId = CHANNEL_ID_COUNTER.addAndGet(1);
+
+    private final IPacketListenerRegistry packetRegistry;
+
+    private final HostAndPort serverAddress, clientAddress;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6033851)